client/signup/steps/domains/index.jsx

```diff
@@ -722,6 +722,8 @@ class DomainsStep extends Component {
 		}
 
 		const { isAllDomains, translate, isReskinned } = this.props;
+		const siteUrl = this.props.selectedSite?.URL;
+		const siteSlug = get( this.props, 'queryObject.siteSlug' );
```

Comment:
We should use the `?` operator instead of `lodash.get` to access the site slug. See p4TIVU-9Bf-p2

Feel free to also tidy up the usage of `get()` on the line below, but the main thing is just not introducing any new lodash usage.

Reply:
> We should use the `?` operator instead of `lodash.get` to access the site slug. See p4TIVU-9Bf-p2
> 
> Feel free to also tidy up the usage of `get()` on the line below, but the main thing is just not introducing any new lodash usage.

Makes sense. I've cleaned this up with 90098a2356